coreos-4/policies/AC-Access_Control/component.yaml

```diff
@@ -463,12 +463,19 @@
 - control_key: AC-4
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - text: |-
-        A control response is planned. Progress can be tracked via:
+        Red Hat CoreOS is not a product that is intended to be used as a data flow control
```

Comment:
What about firewall, router, packet filtering, etc. that happens on the rhcos node?